RecoLocalMuon/GEMSegment/plugins/ME0SegFit.cc

```diff
@@ -343,7 +343,8 @@ ME0SegFit::SMatrixSym12 ME0SegFit::weightMatrix() {
   int row = 0;
   
   for (ME0SetOfHits::const_iterator it = hits_.begin(); it != hits_.end(); ++it) {
-    
+    if (row > 11) break; // temp due to max size of matrix
```

Comment:
This is a temporary fix from @jshlee to avoid a segmentation violation. He or another muon developer will submit a better fix in a subsequent PR.